syncano/models/base.py

```diff
@@ -413,12 +414,22 @@ class CodeBox(Model):
     :ivar name: :class:`~syncano.models.fields.StringField`
     :ivar created_at: :class:`~syncano.models.fields.DateTimeField`
     :ivar updated_at: :class:`~syncano.models.fields.DateTimeField`
+
+    .. note::
+        **CodeBox** has special method called ``run`` which will execute attached source code::
+
+            >>> CodeBox.please.run('instance-name', 1234, payload={'variable_one': 1, 'variable_two': 2})
```

Comment:
What if someone has json already? I think that it would be nice to support both python dictionary and serialized json dicts (don't serialize them again ;) ).